net/v2ray/files/Config.in

```diff
@@ -0,0 +1,149 @@
+menu "V2Ray Configuration"
+	depends on PACKAGE_v2ray-core
+
+choice
+	prompt "JSON Config Support"
+	default V2RAY_JSON_V2CTL
+
+	config V2RAY_JSON_V2CTL
+		bool "Load JSON from V2Ctl"
+
+	config V2RAY_JSON_INTERNAL
+		bool "Load JSON Internally"
+
+	config V2RAY_JSON_NONE
+		bool "None"
+
+endchoice
+
+config V2RAY_EXCLUDE_V2CTL
+	bool "Exclude V2Ctl"
+	depends on V2RAY_JSON_INTERNAL || V2RAY_JSON_NONE
+	default n
+
+config V2RAY_EXCLUDE_ASSETS
+	bool "Exclude geoip.dat & geosite.dat"
+	default n
+
+config V2RAY_COMPRESS_UPX
+	bool "Compress executable files with UPX"
+	default n
+
+choice
+	prompt "Disable Features"
+	default V2RAY_DISABLE_NONE
+
+	config V2RAY_DISABLE_NONE
+		bool "None"
+
+	config V2RAY_DISABLE_CUSTOM
+		bool "Custom"
```

Comment:
I don't know too much about v2ray so this may be a silly question, but is there a reason why someone will want to disable some or all of these features? (And I guess it is not possible to disable these through run-time configuration, that is why it needs to be a build-time decision?)

Most users (I assume) will download the pre-built package (using opkg) and so the binary they will use will have all of these features enabled.

Unless there is a strong need to make these features configurable, I would suggest removing these options to keep things simple.